iothub/device/src/Convention/ClientPropertyCollection.cs

```diff
@@ -28,10 +28,13 @@ public class ClientPropertyCollection : IEnumerable<ClientProperty>
         /// </summary>
         /// <remarks>
         /// Use the <see cref="AddRootProperty(string, object)"/> and/or <see cref="AddComponentProperty(string, string, object)"/> methods
-        /// to add properties into the collection.
+        /// to add properties into the collection. Use the <see cref="AddWritableClientPropertyAcknowledgement(WritableClientPropertyAcknowledgement)"/>
+        /// to add writable property acknowledgements into the collection.
+        /// <para>
         /// This collection can be reported to service using 
         /// <see cref="DeviceClient.UpdateClientPropertiesAsync(ClientPropertyCollection, System.Threading.CancellationToken)"/> or
         /// <see cref="ModuleClient.UpdateClientPropertiesAsync(ClientPropertyCollection, System.Threading.CancellationToken)"/>.
```

Comment:
I wonder if we can avoid linking to the module client method to cut down on the amount of text to avoid that from getting overwhelming. We could simply say "(or corresponding method on the module client)".

Reply:
that sounds like a good idea